### PR TITLE
fix: remove TODOs from examples

### PR DESCRIPTION
**Pull request by drh-determined-ai**
## Description

We should avoid putting `#TODO`s in our example code. Examples should be complete and correct, given whatever state the `determined` code is in. Put another way, examples should show how Determined is awesome, not how it's busted or janky.

Create a Jira issue or GitHub ticket when there's something missing in Determined or if an example can be improved!

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: examples/computer_vision/cifar10_tf_keras/cifar_model.py [hunk 1]
```diff
@@ -12,9 +12,8 @@
 from tensorflow.keras.metrics import categorical_accuracy
 from tensorflow.keras.models import Sequential
 
-# This example is used in some examples with an explicit tf1 image,
+# This example is used in some tests with an explicit tf1 image,
 #  so we keep backwards compatibility.
```
Comment (rb-determined-ai) [inline examples/computer_vision/cifar10_tf_keras/cifar_model.py:16]: I'd drop this whole comment.

It's pretty clear from the code that it's a tf version compatibility thing.

File: examples/tests/test_official.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def test_official(model_def: str, config_file: str) -> None:
         version.parse(tf.__version__) >= version.parse("2.11.0")
         and "gbt_titanic_estimator" in model_def
     ):
-        pytest.skip(f"# TODO [MLG-442], see comment in {model_def}")
```
Comment (rb-determined-ai) [inline examples/tests/test_official.py:66]: tests are ok to have TODOs, but also I like the change here better anyway.

**Review by rb-determined-ai — APPROVED** [review]

File: examples/computer_vision/cifar10_tf_keras/cifar_model.py (hunk 1 above)
Comment (drh-determined-ai) [reply examples/computer_vision/cifar10_tf_keras/cifar_model.py:16]: True, it's clear what the code does in what situations, but without the comment it's not clear why this example gets special treatment.
